### PR TITLE
Running YOLO and Faster RCNN 

### PR DESCRIPTION
**Pull request by victorhg**
Examples of pretrained models to guide ppe categorization

PPE focused pretrained model can serve as benchmark to multiple item detection for our own model training

### CODE REVIEW

**Comment by caliskate** [comment]
looks good thanks, 
WIP: add custom dataset, data preprocessing, confidence curves, confusion matrix, validation loss charts, missing PPE plot

**Review by caliskate — APPROVED** [review]
